reference/v8.json

```diff
@@ -595,10 +595,11 @@
       "property-function": true,
       "values": [
         "map",
-        "viewport"
+        "viewport",
+        "auto"
       ],
-      "default": "viewport",
-      "doc": "Orientation of icon when map is rotated.",
+      "default": "auto",
```

Comment:
@lucaswoj do the changes to the doc strings in 296d335 address your concerns about the lack of explanation of `auto`?